std_msgs/Header header
pick_and_place/DetectedObject[] detected_objects
